source/frontend/src/components/maps/hooks/useActiveFeatureLayer.tsx

```diff
@@ -152,7 +152,9 @@ const useActiveFeatureLayer = ({
           : DistrictCodes.Unknown,
         DISTRICT_NAME: district.DISTRICT_NAME ?? 'Cannot determine',
       };
-      activeFeatureLayer?.addData(activeFeature);
+      if (activeFeature.geometry.type === 'Polygon') {
```

Comment:
PSP-5863 basically this code is trying to add a highlighted feature even if no feature data is available, that results in a placeholder marker with a lat/lng set to the click location. Since we don't have a marker set up for that it displays as a missing placeholder image.